index.html

```diff
@@ -21,14 +22,118 @@ <h2 class="hsecond">I am a Software Developer</h2>
       </ul>
       <img class="stetescope-icone" alt="stetescope" src="images/design.svg">
     </section>
+    <section class="recent-work">
+      <h2 class="title-work">My Recent Work</h2>
+      <img class="pro-img" alt="project image" src="images/Imgplaceholder.png">
+      <h2 class="stories">Multi-Post Stories</h2>
+      <p class="about-stories">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+        been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+        text.</p>
+      <ul class="cards">
+        <li>CSS</li>
+        <li>HTML</li>
+        <li>Bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <a class="link-pro" href="#">
+        <div class="see-pro">See Project</div>
+      </a>
+      <div class="second-project">
+        <h2 class="project-heading">Profesional Art<br>
+          Printing Data</h2>
+        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+        <ul class="cards-project2">
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <a class="link-project2" href="#">
+        <div class="see-project2">See Project</div>
+      </a>
+      <div class="second-project">
+        <h2 class="project-heading">Profesional Art<br>
+          Printing Data</h2>
+        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+        <ul class="cards-project2">
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <a class="link-project2" href="#">
+        <div class="see-project2">See Project</div>
+      </a>
+      <div class="second-project">
+        <h2 class="project-heading">Profesional Art<br>
+          Printing Data</h2>
+        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+        <ul class="cards-project2">
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <a class="link-project2" href="#">
+        <div class="see-project2">See Project</div>
+      </a>
+      <div class="second-project">
+        <h2 class="project-heading">Profesional Art<br>
+          Printing Data</h2>
+        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+        <ul class="cards-project2">
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <a class="link-project2" href="#">
+        <div class="see-project2">See Project</div>
+      </a>
```

Comment:
- [ ] Please center the text inside each one of work cards to match the given design.
- [ ] It would be better semantically if you make `see project` as a <button> tag instead of making it inside `<div>` tag wrapped by `<a>` tag.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/159141273-dedb70a0-a822-4387-a6d9-a27385a308b0.png) | ![image](https://user-images.githubusercontent.com/68796232/159141288-e0cf5f01-50f5-4777-b415-4c60dfa67808.png) |